src/app/components/Book/Book.jsx

```diff
@@ -63,21 +65,19 @@ const Book = ({ pick, isJsEnabled }) => {
         href={catalogUrl}
         className="catalog-url"
         onClick={() => gaEvent('Book')}
-        aria-label={`Request Book: ${book.title}`}
       >
         <BookIcon width="32px" height="32px" ariaHidden />
-        <span>{config.requestUrlsText.catalog}</span>
+        <span aria-label={`Request Book: ${book.title}`}>{config.requestUrlsText.catalog}</span>
```

Comment:
Are we using `aria-label` or `aria-labelledby` here for the book titles? I see `aria-labelledby` used for tags, but not in the book items themselves. Is that correct?

Reply:
Yeah, using `aria-label` to be more specific for the two request buttons.

Using `aria-labelledby` for the book cards since I'm linking the h3 id to the book card.